lib/Sema/CSDiagnostics.cpp

```diff
@@ -639,11 +649,10 @@ Optional<Diag<Type, Type>> GenericArgumentsMismatchFailure::getDiagnosticFor(
     return diag::cannot_convert_condition_value;
   case CTP_WrappedProperty:
     return diag::wrapped_value_mismatch;
-  case CTP_ComposedPropertyWrapper:
-    return diag::composed_property_wrapper_mismatch;
 
   case CTP_ThrowStmt:
   case CTP_ForEachStmt:
+  case CTP_ComposedPropertyWrapper:
```

Comment:
Nit: Maybe for a follow up PR, but since now this is tracked using a custom locator `CTP_ComposedPropertyWrapper` is not used anymore(or there is some other place?). 
If is indeed not used, maybe we can remove it @xedin?